lib/demographic_details_view.dart

```diff
@@ -0,0 +1,902 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/framework.dart';
+import 'package:registration_client/data/models/field.dart';
+import 'package:registration_client/data/models/screen.dart';
+import 'package:registration_client/provider/global_provider.dart';
+
+import 'package:provider/provider.dart';
+
+final Color secondaryColor = Colors.blue;
+
+final Color primaryColor = Colors.blue;
+final specificJsonData = {
+  "fields": [
```

Comment:
let's not push the changes added for local testing

Reply:
Removed local testing code